kpops/cli/main.py

```diff
@@ -198,13 +198,37 @@ def generate(
     defaults: Path = DEFAULT_PATH_OPTION,
     config: Path = CONFIG_PATH_OPTION,
     verbose: bool = typer.Option(False, help="Enable verbose printing"),
+    template: bool = typer.Option(False, help="Run helm template"),
+    steps: Optional[str] = PIPELINE_STEPS,
+    api_version: Optional[str] = typer.Option(
+        None, help="Kubernetes api version used for Capabilities.APIVersions"
+    ),
+    ca_file: Optional[str] = typer.Option(
+        None, help="Verify certificates of HTTPS-enabled servers using this CA bundle"
+    ),
+    cert_file: Optional[str] = typer.Option(
+        None, help="Identify HTTPS client using this SSL certificate file"
+    ),
 ):
     pipeline_config = create_pipeline_config(config, defaults, verbose)
     pipeline = setup_pipeline(
         pipeline_base_dir, pipeline_path, components_module, pipeline_config
     )
     pipeline.print_yaml()
 
+    if template:
+        steps_to_apply = get_steps_to_apply(pipeline, steps)
+        for component in steps_to_apply:
+            component.template(api_version, ca_file, cert_file)
+    elif cert_file or ca_file or api_version or steps:
```

Comment:
I think `steps` isn't exclusive to `--template`

Reply:
It is. It wasn't in generate's flags before.

This is the "old" part of `generate`:

``` python
pipeline_config = create_pipeline_config(config, defaults, verbose)
pipeline = setup_pipeline(
    pipeline_base_dir, pipeline_path, components_module, pipeline_config
)
pipeline.print_yaml()
```

Reply:
you're right